views/artist-search-results.hbs

```diff
@@ -0,0 +1,7 @@
+{{#each ArtistList}}
+  <div style="border: 1px solid black; padding: 20px">
+    <p>{{this.name}}</p>
+    <p><img src="{{this.images.0.url}}" height=300px></p>
+    <button><a href="/albums/{{this.id}}">View Albums</a></button>
```

Comment:
no need to wrap the img with the paragraph, and the link (anchor) with the button